File: neatformcore/src/main/java/com/nerdstone/neatformcore/domain/model/NFormViewData.java
package com.nerdstone.neatformcore.domain.model;

import java.io.Serializable;
import java.util.Map;

public class NFormViewData implements Serializable {

  private String name;
  private Object value;
  private String label;
  private Map<String, Object> metadata;

  public String getName() {
    return name;
  }

  public void setName(String name) {
    this.name = name;
  }

  public Object getValue() {
    return value;
  }

  public void setValue(Object value) {
    this.value = value;
  }

  public String getLabel() {
    return label;
  }

  public void setLabel(String label) {
    this.label = label;
  }

  public Map<String, Object> getMetadata() {
    return metadata;
  }

  public void setMetadata(Map<String, Object> metadata) {
    this.metadata = metadata;
  }
}

File: neatformcore/src/main/java/com/nerdstone/neatformcore/domain/model/NFormViewData.kt
package com.nerdstone.neatformcore.domain.model

import android.arch.persistence.room.Entity

import java.io.Serializable

@Entity
class NFormViewData : Serializable {
    var name: String? = null
    var value: Any? = null
    var label: String? = null
    var metadata: Map<String, Any>? = null
}
